tests/foreman/ui/test_host.py

```diff
@@ -367,6 +367,154 @@ def test_positive_create_libvirt(self):
             self.assertIsNotNone(search)
 
     @run_only_on('sat')
+    @stubbed
+    @tier3
+    def test_positive_create_baremetal_bios(self):
```

Comment:
I think user should be able to pass the pxe BIos/UEFI loader from CLI and api but worth checking the exact option.